tests/sqllogictests/suites/mode/standalone/explain/join.test

```diff
@@ -825,19 +829,15 @@ EvalScalar
     ├── probe keys: [replace(t1.ext (#1), '33', '44')]
     ├── filters: []
     ├── estimated rows: 0.00
-    ├── Filter(Build)
-    │   ├── output columns: [t2.wms_id (#4)]
-    │   ├── filters: [is_true(substr(t2.wms_id (#4), 1) = 'R006')]
-    │   ├── estimated rows: 0.00
-    │   └── TableScan
-    │       ├── table: default.default.t2
-    │       ├── output columns: [wms_id (#4)]
-    │       ├── read rows: 0
-    │       ├── read bytes: 0
-    │       ├── partitions total: 0
-    │       ├── partitions scanned: 0
-    │       ├── push downs: [filters: [is_true(substr(t2.wms_id (#4), 1) = 'R006')], limit: NONE]
-    │       └── estimated rows: 0.00
+    ├── TableScan(Build)
+    │   ├── table: default.default.t2
+    │   ├── output columns: [wms_id (#4)]
+    │   ├── read rows: 0
+    │   ├── read bytes: 0
+    │   ├── partitions total: 0
+    │   ├── partitions scanned: 0
+    │   ├── push downs: [filters: [], limit: NONE]
+    │   └── estimated rows: 0.00
```

Comment:
Seems the filter `substr(t2.wms_id, 1) = 'R006'` is failed to push down?

Reply:
Yes, fixed